.github/actions/slack-status/action.yml

```diff
@@ -0,0 +1,37 @@
+name: "Send Slack Status Notification"
+description: Slack Status Notification (replacement for circleci_slack_status orb).
+
+inputs:
+  message:
+    description: The Markdown message contents
+    required: true
+  slack-webhook-url:
+    description: Slack channel webhook URL to post the message
+    required: true
+  run-url:
+    description: Self-referencing RUN
+    required: true
+  block-color:
+    description: HEX color code, use `ed5c5c` for failure and `1cbf43` for success status
+    required: false
+    default: 1cbf43
+
+runs:
+  using: "composite"
+  steps:
+    - name: Send Slack Notification
+      uses: slackapi/slack-github-action@v1.23.0
```

Comment:
Should this be pinned to a SHA?

Reply:
I think security is supposed to have a bot that'll come through and do this, but I can certainly fix it now!